fv3core/stencils/d_sw.py

```diff
@@ -323,53 +406,37 @@ def heat_source_from_vorticity_damping(
         if __INLINED((d_con > dcon_threshold) or do_skeb):
             with horizontal(region[local_is : local_ie + 1, local_js : local_je + 1]):
                 heat_source_total = heat_source_total + heat_source
-                # do_skeb could be renamed to calculate_dissipation_estimate
-                # when d_sw is converted into a D_SW object
+                # TODO: do_skeb could be renamed to calculate_dissipation_estimate
                 if __INLINED(do_skeb == 1):
                     dissipation_estimate = diss_e - dampterm
     with computation(PARALLEL), interval(...):
         if damp_vt > 1e-5:
             with horizontal(region[local_is : local_ie + 1, local_js : local_je + 2]):
-                u = u + vt
+                u = u + damped_u_dx
             with horizontal(region[local_is : local_ie + 2, local_js : local_je + 1]):
-                v = v - ut
-
-
-def update_ke(
-    ke: FloatField,
-    u: FloatField,
-    v: FloatField,
-    ub: FloatField,
-    vb: FloatField,
-    ut: FloatField,
-    vt: FloatField,
-    dt: float,
-):
-    with computation(PARALLEL), interval(...):
-        ke = ke_from_bwind(ke, ub, vb)
-        ke = all_corners_ke(ke, u, v, ut, vt, dt)
+                v = v - damped_v_dy
 
 
 def update_horizontal_vorticity(
     u: FloatField,
     v: FloatField,
-    ut: FloatField,
-    vt: FloatField,
+    v_dy: FloatField,
+    u_dx: FloatField,
     dx: FloatFieldIJ,
     dy: FloatFieldIJ,
     rarea: FloatFieldIJ,
     vorticity: FloatField,
 ):
     with computation(PARALLEL), interval(...):
-        vt = u * dx
-        ut = v * dy
+        u_dx = u * dx
+        v_dy = v * dy
     # TODO(rheag). This computation is required because
     # ut and vt are API fields. If the distinction
     # is removed, so can this computation.
     # Compute the area mean relative vorticity in the z-direction
     # from the D-grid winds.
     with computation(PARALLEL), interval(...):
-        vorticity = rarea * (vt - vt[0, 1, 0] - ut + ut[1, 0, 0])
+        vorticity = rarea * (u_dx - u_dx[0, 1, 0] - v_dy + v_dy[1, 0, 0])
```

Comment:
This is `wk` in fortran, which is volume_mean_relative_vorticity

Reply:
Yes, sounds good. Fortran reference is [here](https://github.com/VulcanClimateModeling/fv3gfs-fortran/blob/e42cd14e08d6b08910fdec7912a3cc4377145e8a/FV3/atmos_cubed_sphere/model/sw_core.F90#L1370).